test/mocha/10-api.js

```diff
@@ -44,4 +44,76 @@ describe('readiness', () => {
     result.dependencies.ready1.ready.should.equal(true);
     result.dependencies.ready2.ready.should.equal(false);
   });
+  it('should throw error if readiness protocol is not supported',
+    async function() {
+      const dependency = {
+        type: 123,
+        parameters: {
+          url: 'urn:zcap:xyz'
+        }
+      };
+
+      const result = await readiness._validateHttpGet(
+        {name: 'ready3', dependency});
+
+      should.exist(result);
+      result.valid.should.equal(false);
+      const {error: err} = result;
+      should.exist(err);
+      err.name.should.equal('DataError');
+      err.message.should.equal('Readiness dependency "ready3" is invalid.');
+      err.details.httpStatusCode.should.equal(500);
+      err.cause.message.should.equal('HTTP readiness protocol "urn:" not ' +
+      'supported. Supported protocols are: https:, http:');
+    });
+  it('should throw error if readiness check type is already registered',
+    async function() {
+      let result;
+      let err;
+      try {
+        result = await readiness.registerType(
+          {type: 'httpGet', run: async () => {}, validate: async () => {}});
+      } catch(e) {
+        err = e;
+      }
+
```

Comment:
inconsistent use of white space.  There is white space here after try/catch but not in the test starting on L87.  Please try to be consistent throughout.

Reply:
addressed 4b29d14